src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import dukeclass.Parser;
+import dukeclass.Storage;
+import dukeclass.TaskList;
+import dukeclass.Ui;
```

Comment:
When we want to import more than three components from the same package, is it better to use "import dukeclass.*"?

Reply:
Hi, I don't think using wildcard imports is better. The following line can be found in the CS2103T Java coding standard:
> Imported classes should always be listed explicitly.

Reply:
Sorry, I think I have remembered it wrongly :( Thanks!